checker/values.ml

```diff
@@ -385,9 +385,6 @@ module IntMap = Map.Make(IntOrd)
 
 let dyn_table : value IntMap.t ref = ref IntMap.empty
 
-let register_dyn name t =
-  dyn_table := IntMap.add name t !dyn_table
-
```

Comment:
What's going on here? We only expose the accessor function below, so the `dyn_table` will always be empty.

Reply:
This is a function whose use is purely speculative. It'd make possible to register data representation dynamically, e.g. if one wants to extend `votour` in interactive use. It's not used yet.

Reply:
Why not wait to have actual use cases to introduce the code?

Reply:
Yes, you can remove it. There is no way to link code into votour currently anyways.